constellations/gemini-admin/src/announce/announce.sass

```diff
@@ -0,0 +1,71 @@
+@import "../_.constants.sass"
+
+#view-announce
+    width: 100%
+    height: 400px
+    padding: 0 $spacing2x
+    box-sizing: border-box
+    color: $colorGrayDark
+    font-family: $fontFamilyRoboto
+
+    & h1
+        margin: $spacing4x 0
+        line-height: $lineHeight3
+        font-size: $fontSize3
+    
+    & ul
+        margin: 0
+        padding: 0
+        list-style: none
+
+    & .announce-list-row
+        display: block
+        position: relative
+        padding: $spacing2x 0
+        border-bottom: 1px solid $colorGrayLight
+        
+        display: flex
+        flex-direction: row
+        align-items: center
```

Comment:
this has display properties of both block and flex. If it's only display: flex, make sure the CSS properties go on the top for proper CSS ordering